float32 roll_integ		# roll rate integrator
float32 pitch_integ	# pitch rate integrator
float32 yaw_integ		# yaw rate integrator

float32 additional_integ1	# FW: wheel rate integrator
